src/lib.rs

```diff
@@ -8,7 +8,7 @@
 #![allow(clippy::large_enum_variant, clippy::derive_partial_eq_without_eq)]
 #![allow(rustdoc::bare_urls)]
 #![forbid(unsafe_code)]
-#![doc(html_root_url = "https://docs.rs/ibc-proto/0.21.0")]
+#![doc(html_root_url = "https://docs.rs/ibc-proto/0.22.0")]
```

Comment:
Note that the crate version in Cargo.toml is already `0.22.0` (done in a previous PR).